flow/probes/ebpf.go

```diff
@@ -132,14 +132,14 @@ func (p *EBPFProbe) run() {
 		case now = <-updateNow.C:
 		default:
 			if statsMap := p.module.Maps["stats_map"]; statsMap != nil {
-				var statsKey uint32
-				var statsVal int64
+				var dropKey uint32
+				var dropValue int64
 
-				if found, err := statsMap.Get(statsKey, &statsVal); err == nil && found {
-					if statsVal > 0 {
-						p.Ctx.Logger.Warningf("flow table overflow, %d flows were dropped from kernel table", statsVal)
+				if found, err := statsMap.Get(dropKey, &dropValue); err == nil && found {
```

Comment:
does dropKey not need to be initialized?

Reply:
it is, golang initializes it to 0